src/coreclr/jit/compiler.h

```diff
@@ -2928,6 +2928,7 @@ class Compiler
     bool gtComplexityExceeds(GenTree* tree, unsigned limit);
 
     GenTree* gtReverseCond(GenTree* tree);
+    GenTree* gtSwapRelop(GenTree* tree);
```

Comment:
We have a number of functions that call `GenTree::SwapRelop` and/or `gtReverseCond` today. Can we either update those as well or log an issue tracking updating them to be consistent?

Reply:
Seems fair to update all cases.

Reply:
Turns out there was only one other use of `GenTree::SwapRelop` that gets used to change an oper.